bundles/org.openhab.binding.ferroamp/README.md

```diff
@@ -0,0 +1,239 @@
+## Ferroamp Binding
+
+The Ferroamp binding is used to get live data from Ferroamp EnergyHub
+
+The Ferroamp binding is compatible with EnergyHub Wall and EnergyHub XL, and connects to your local EnergyHub via LAN.
+Data and commands are received/sent using MQTT where the user connects to the MQTT broker residing on the EnergyHub.
+
+*note* Contact Ferroamp support to enable MQTT in the EnergyHub and to get the Username and Password:
+
+https://ferroamp.com/om-ferroamp/
+
+## Supported Things
+
+The binding retrieves data from the different parts of the Ferroamp EnergyHub such as:
+
+- `ehub`: EnergyHub Wall and EnergyHub XL.
+- `sso`: Solar string optimizer.
+- `eso`: Bidirectional DC/DC converter for connection of battery.
+- `esm`: Energy Storage Module.
+
+## Discovery
+
+Discovery is not supported.
+
+## Thing Configuration
+
+The following configuration parameters are available.
+
+| Name            | Type    | Description                              | Default | Required | Advanced |
+|-----------------|---------|------------------------------------------|---------|----------|----------|
+| hostName        | text    | Hostname or IP address of the device     | N/A     | yes      | no       |
+| userName        | text    | Username to access the device            | N/A     | yes      | no       |
+| password        | text    | Password to access the device            | N/A     | yes      | no       |
+| hasBattery      | boolean | Is there a battery connected or not?     | N/A     | no       | yes      |
+| ssoS0           | boolean | Is there a first SSO unit connected?     | N/A     | no       | no       |
+| ssoS1           | boolean | Is there a second SSO unit connected?    | N/A     | no       | no       |
+| ssoS2           | boolean | Is there a third SSO unit connected?     | N/A     | no       | no       |
+| ssoS3           | boolean | Is there a fourth SSO unit connected?    | N/A     | no       | no       |
```

Comment:
Is there a system limit to 4 SSO's ? Did you consider to model this parameter as a single number:  0-9 representing the amount of SSO's this way it would be more dynamic?

Reply:
Actually, there is a limit of 64 SSO's for a huge system.
But I think that this binding  will be for smaller properties, like ordinary houses.
Each SSO can be responible for 8kW and most houses in at least Sweden has a maximum incoming/outgoing effect of like 30kW ( 25A ).